File: XZKit/Objective-C/Foundation/XZKitDEBUG.m
//
//  XZKitDEBUG.m
//  XZKit
//
//  Created by Xezun on 2021/2/7.
//  Copyright © 2021 Xezun Inc. All rights reserved.
//

#define XZKIT_PROTECTED
#import "XZKitDEBUG.h"

bool XZKitDebugMode = NO;

@implementation NSObject (XZKitDEBUG)

+ (void)load {
    XZKitDebugMode = (bool)[NSProcessInfo.processInfo.arguments containsObject:@"-XZKitDEBUG"];
}

@end

File: XZKit/Sources/Swift/XZKitDEBUG.swift
//
//  XZKitDEBUG.swift
//  XZKit
//
//  Created by Xezun on 2021/2/28.
//

import Foundation

/// 标识当前是否处于调试模式。
public let isDebugMode = ProcessInfo.processInfo.arguments.contains("-XZKitDEBUG")
